app/workers/migrate_data_to_work_field_worker.rb

```diff
@@ -0,0 +1,13 @@
+# NOTE: we can remove this once the DUS using it finished
+class MigrateDataToWorkFieldWorker
+  include Sidekiq::Worker
+
+  def perform(profile_id)
+    profile = Profile.find_by(id: profile_id)
+    return unless profile
+
+    work_info = profile.employment_title
+    work_info << " at #{profile.employer_name}" if profile.employer_name.present?
+    profile.update(work: work_info)
```

Comment:
It's an unvalidated field.

Reply:
Is there any other way it could fail validations? For example, a validation that's added after the data got into a state that the new validation considers invalid?

Reply:
> Is there any other way it could fail validations? 

It's hard to say "no" to this since adding a validation for existing data could always lead to invalid records. 

Tbh for the purpose of this DUS I don't really care about that so I'd rather just use `update_column` to make sure we migrate the work data.